spec/features/user_can_login_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.feature "Login", type: :feature do
+  scenario "allows a User to login" do
+    visit '/'
+    expect(page).to have_content("You are a Guest, please login")
```

Comment:
If this is one of the feature tests you've mentioned are currently failing, it might be because in the html.erb file, it says: "You are a guest, please login or signup" (i.e. guest is all in lower case, hence it might not find 'Guest' if it's case-sensitive).

Reply:
Agreed! I was too lazy this weekend to fix these, but yeah i think it's as simple as that luckily. thank you!!